### DISCUSSION TITLE
Unable to contribute to documentation/docs of SvelteKit

### DISCUSSION THREAD

**Discussion by Devr-pro** [sveltejs/kit#8771] [Q&A] [answered]
Hey there! I am very excited and happy about what the Svelte and SvelteKit projects are doing, and I really to contribute to it
I am getting this weird situation where I can't access `modules.md` in  `documentation/docs/50-reference/` , It only shows title and heading of that md ,Link--> (https://github.com/sveltejs/kit/tree/master/documentation/docs/50-reference/30-modules.md)
Is this expected or am going wrong? Thanks! (Sorry if this sounds silly 😅,this is my first time to contribute in SvelteKit)

**Comment by teemingc** [comment, marked as answer]
The docs for those are generated from the comments in any of the types files here: https://github.com/sveltejs/kit/tree/master/packages/kit/types

You can edit the comments in `index.d.ts`, `ambient.d.ts`, or `/synthetic/*`.

This isn't mentioned anywhere, so don't worry about it! Had to do a bit of digging myself to figure this out.

**Reply by Devr-pro** [reply]
Oh, I see that's why I cant find it, Thanks, appreciate your reply!
